### PR TITLE
Icons for several links on task editor

### PR DESCRIPTION
**Pull request by M-Falken**
Replace the group pic by the fa users icon

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

File: inc/commonitiltask.class.php [hunk 1]
```diff
@@ -1531,10 +1531,8 @@ function tasktemplate_update(value) {
                                   'url'       => $CFG_GLPI["root_doc"]."/ajax/planningcheck.php"];
       User::dropdown($params);
 
-      echo " <a href='#' onClick=\"".Html::jsGetElementbyID('planningcheck'.$rand).".dialog('open'); return false;\">";
-      echo "&nbsp;<img src='".$CFG_GLPI["root_doc"]."/pics/reservation-3.png'
-             title=\"".__s('Availability')."\" alt=\"".__s('Availability')."\"
-             class='calendrier'>";
+      echo " <a href='#' title=\"".__s('Availability')."\" onClick=\"".Html::jsGetElementbyID('planningcheck'.$rand).".dialog('open'); return false;\">";
+      echo "&nbsp;<i class='far fa-calendar-alt'></i>";
```
Comment (trasher) [inline inc/commonitiltask.class.php]: For accessibility, add `"<span class='sr-only'>".__('Availability')."</span>"` (will replace the `img@alt`.

File: inc/commonitiltask.class.php [hunk 2]
```diff
@@ -1543,7 +1541,7 @@ class='calendrier'>";
                                     ['title'  => __('Availability')]);
 
       echo "<br />";
-      echo Html::image($CFG_GLPI['root_doc']."/pics/group.png")."&nbsp;";
+      echo "<i class='fas fa-users'></i>&nbsp;";
```
Comment (trasher) [inline inc/commonitiltask.class.php]: Since this one is decorative only, add a ` aria-hidden='true'`

Comment (M-Falken) [reply inc/commonitiltask.class.php]: Thanks, I just added `aria-hidden` as the "true" is not mandatory

File: inc/commonitiltask.class.php (hunk 1 above)
Comment (M-Falken) [reply inc/commonitiltask.class.php]: Thanks, done

**Comment by trasher** [comment]
See https://github.com/glpi-project/glpi/pull/5249

**Pull request closed without merging**